libs/cosmos-sdk/store/mpt/types/wrap_rocksdb.go

```diff
@@ -56,6 +57,16 @@ func (db *WrapRocksDB) Compact(start []byte, limit []byte) error {
 	return nil
 }
 
+func (db *WrapRocksDB) NewBatchWithSize(size int) ethdb.Batch {
+	panic(fmt.Errorf("not support NewBatchWithSize"))
+	return nil
```

Comment:
It seems the return is unnecessary.

Reply:
This mpt code adapt code, we have not plan to support mpt for lastest ethereum code . yes it's useless code , i will remove it later